app/scripts/migrations/091.test.js

```diff
@@ -0,0 +1,149 @@
+import { migrate, version } from './091';
+
+jest.mock('uuid', () => {
+  const actual = jest.requireActual('uuid');
+
+  return {
+    ...actual,
+    v4: jest.fn(),
+  };
+});
+
+describe('migration #91', () => {
+  it('should update the version metadata', async () => {
+    const oldStorage = {
+      meta: {
+        version: 90,
+      },
+      data: {},
```

Comment:
Should probably audit this file to ensure the right version is being applied throughout. in some cases version is 90 and in others its 88 in old data and we aren't validating that 91 is used anywhere (we use the exported version which is currently 89)

Reply:
fixed in 5c93be014c